site/docs/advanced/business.md

```diff
@@ -0,0 +1,141 @@
+# Telegram Business
+
+Telegram Business allows your private chat with another (human) user to be managed by a bot
+This includes sending and receiving messages on your behalf.
+Typically, this is useful if you run your business on Telegram, and that other user is a customer.
```

Comment:
```suggestion
Typically, this is useful if you run your business on Telegram, and that other users are customers.
```

Reply:
We talk about the chat between the reader and a different user. This sentence refers to that other user in said chat, not to any number of people.